packages/flutter/lib/src/widgets/text_selection_toolbar_layout_delegate.dart

```diff
@@ -0,0 +1,86 @@
+import 'dart:math' as math;
```

Comment:
Is `widgets` the right place for this?  I could also package this as a widget or something.